res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe

```diff
@@ -0,0 +1,252 @@
+operand {
```

Comment:
if this is generated from tflchef-reverse, please add a comment from and how this was generated.

Reply:
I changed recipe to single digit value and also add a comment how this recipe was generated.
Please take a look =)